src/cr/cube/matrix/cubemeasure.py

```diff
@@ -312,6 +312,15 @@ def columns_base(self):
         """
         return np.sum(self._unweighted_counts, axis=1)
 
+    @lazyproperty
+    def columns_pruning_base(self):
+        """1D np.int64 ndarray of unweighted-N for each matrix column.
+
+        These values include both the selected and unselected counts of the MR rows
+        dimension.
+        """
+        return np.sum(self._unweighted_counts, axis=(0, 1))
```

Comment:
This seems odd, you're adding across subvariables dimension. Maybe a better choice here would be to use `not all` across that (zeroth) subvariables dimension. It's gonna amount to the same thing, since we're only using it to show/hide values, but maybe worth considering.

Reply:
Yeah, that's an interesting point. I was blowing through all this pretty fast just because there was so much ground to cover, but in retrospect I'm inclined to think that this property should be something more like `.pruning_mask` and return an array of boolean rather than integers that are elsewhere compared to zero to determine that boolean value. That would be better because these counts would never occur outside this method where they could be misinterpreted.

I don't have time for that change today, but maybe I'll sneak it in next time I'm in here.